ui/src/main/java/com/andersen/ui/controller/BookingController.java

```diff
@@ -65,6 +86,13 @@ public ResponseEntity<List<BookingDto>> getBookingsByUser(@PathVariable Long use
         }
```

Comment:
it's not a bad request, it's internal server error. bad request should be in case of validation errors, so divide it